requirements.txt

```diff
@@ -1 +1 @@
-purefusion>=1.0,<2.0
+purefusion>=1.x.x,<2.0  # TODO change as soon as we know the specific version !
```

Comment:
**Don't forget to apply** and **to correct** this suggestion before merging this Pull Request!
```suggestion
purefusion>=1.1.0,<2.0
```
The feature introduced in this PR isn't backward compatible with old versions of purefusion Python SDK. Forward compatibility remains.

Reply:
It's `1.0.11` in the end.